apps/api/src/app/events/e2e/digest-events.e2e.ts

```diff
@@ -1226,9 +1228,16 @@ describe('Trigger event - Digest triggered events - /v1/events/trigger (POST)',
     const mergedJobs = jobs.filter((elem) => elem.status !== JobStatusEnum.DELAYED);
     expect(mergedJobs.length).to.eql(9);
 
-    const delayedJobUpdateTime = delayedJobs[0].updatedAt;
+    let delayedJobUpdateTime = delayedJobs[0].updatedAt;
     expect(delayedJobUpdateTime).to.be.ok;
 
+    let delayed = delayedJobs[0];
+    do {
+      delayed = (await jobRepository.findById(delayedJobs[0]._id)) as JobEntity;
+      delayedJobUpdateTime = delayed.updatedAt;
+      await promiseTimeout(100);
+    } while (delayed.status !== JobStatusEnum.COMPLETED);
```

Comment:
had to fix this failing test like this, super hacky but couldn't find a better way for now... @p-fernandez fyi